include/fir/Attribute.h

```diff
@@ -1,16 +1,10 @@
-// Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+//===-- include/fir/Attribute.h ---------------------------------*- C++ -*-===//
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+//----------------------------------------------------------------------------//
```

Comment:
Shouldn't this footer line be "//===-- ... --===//" instead of all dashes?  I think that's what the MLIR sources have, IIRC.

Reply:
They do. I just checked. :)

Reply:
LLVM itself uses all dashes  on the trailing line. That's what we put into the rest of f18.

Reply:
Not seeing that.  Taking a random file ... llvm/lib/MCA/InstrBuilder.cpp ... it is using the 3-equals format.

Reply:
Oh my.

Reply:
Did you add your latest to this pull request?